main.libsonnet

```diff
@@ -0,0 +1,25 @@
+{
+  File(name, content=""):: {
+    local file = self,
+
+    apiVersion: "jsonnet-filer.zeet.co/v1alpha1",
```

Comment:
k8s style APIs usually don't have dashes in the prefix. https://docs.openshift.com/container-platform/4.12/rest_api/index.html#api-index 

dashes are allowed in the `/path` part.

Reply:
You piqued my curiosity, so I went digging for the spec:

> Group names must be lower case and be valid DNS subdomains.

https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md?plain=1#L120

Reply:
> DNS names can contain only alphabetical characters (A-Z), numeric characters (0-9), the minus sign (-), and the period (.). Period characters are allowed only when they are used to delimit the components of domain style names

Reply:
that is technically correct. I mentioned it because I haven't seen any group name using dashes in the subdomain in the wild. The only dashes I've seen are in the root domain itself like `cert-manager.io`